File: DiemdanhHocVienAndroid2/app/src/main/java/com/example/diemdanhhocvienandroid2/models/Student.java
package com.example.diemdanhhocvienandroid2.models;

public class Student {
    private int id;
    private String lastName;
    private String firstName;
    private String holyName;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getHolyName() {
        return holyName;
    }

    public void setHolyName(String holyName) {
        this.holyName = holyName;
    }

    public String getBOD() {
        return BOD;
    }

    public void setBOD(String BOD) {
        this.BOD = BOD;
    }

    public String getNumPhone() {
        return numPhone;
    }

    public void setNumPhone(String numPhone) {
        this.numPhone = numPhone;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public int getParentId() {
        return parentId;
    }

    public void setParentId(int parentId) {
        this.parentId = parentId;
    }

    public int getClassId() {
        return classId;
    }

    public void setClassId(int classId) {
        this.classId = classId;
    }

    private String BOD;
    private String numPhone;
    private String email;
    private int parentId;
    private int classId;
}

File: DiemdanhHocvien/DataAccess/Student.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace DiemdanhHocvien.DataAccess
{
    public class Student
    {
        [Key]
        public int id { get; set; }
        [Required]
        public string lastName { get; set; }
        [Required]
        public string firstName { get; set; }
        [Required]
        public string holyName { get; set; }
        [Required]
        public DateTime BOD { get; set; }
        public string numPhone { get; set; }
        public string email { get; set; }
        public int parentId { get; set; }
        public int classId { get; set; }
    }
}